validator.js

```diff
@@ -450,7 +450,26 @@
     };
 
     validator.isDate = function (str) {
-        return !isNaN(Date.parse(str));
+        var rawDate = new Date(str);
+        var normalizedDate = new Date(rawDate.getTime() + rawDate.getTimezoneOffset()*60000);
```

Comment:
Fair enough. I was wondering more about the `60000` but I see now that `getTimezoneOffset()` returns an offset in minutes.